hms-api/serverless.local.yml

```
@@ -55,6 +57,8 @@ provider:
     PARTICIPANT_TABLE: participant
     PARTICIPANT_BY_HACKATHON_ID_INDEX: participant-by-hackathonId
     PARTICIPANT_BY_HACKATHON_ID_USER_ID_INDEX: participant-by-hackathonId-userId
+    COMMENT_TABLE: ideaComment
+    COMMENT_BY_IDEA_ID_INDEX: ideaComment-by-ideaId
```

Just for consistency I'd do...

```suggestion
    IDEA_COMMENT_TABLE: ideaComment
    IDEA_COMMENT_BY_IDEA_ID_INDEX: ideaComment-by-ideaId
```

---

be sure to do a global search for the names and replace them